### PR TITLE
Added non-breaking space character support for any content

### PR DESCRIPTION
**Pull request by AlexandrRzz**
Related PR: https://github.com/area9innovation/innovation/pull/2432

https://trello.com/c/B0wU7ebi/13862-a-non-breakable-space-is-not-working-in-a-math-font-in-the-reader

### CODE REVIEW

**Comment by AlexandrRzz** [comment]
[A non-breakable space is not working in a math font in the reader](https://trello.com/c/B0wU7ebi/13862-a-non-breakable-space-is-not-working-in-a-math-font-in-the-reader)

File: lib/form/paragraph3.flow [hunk 1]
```diff
@@ -374,14 +374,16 @@ renderParagraphEx(
 			default: false;
 		};
 		lastChr = strsubsmart(getWordText(lastElement(group.words, emptyParaWord)), -1, 0);
-		lastChrIsMath = lastChr != "" && strContains("+-=×/", lastChr);
+		firstChr = strsubsmart(getWordText(word), 0, 1);
+		isNonBreakableSpace = lastChr == fromCharCode(0x00a0) || firstChr == fromCharCode(0x00a0);
```
Comment (ykuzmenko73) [inline lib/form/paragraph3.flow:378]: make it as global constant

**Review by ykuzmenko73 — COMMENTED** [review]
LGTM

File: lib/form/paragraphtypes.flow [hunk 2]
```diff
@@ -3,6 +3,8 @@ import ds/list;
 import form/paragraphalignments;
 
 export {
+	nonBreakableSpaceCode = 0x00a0;
```
Comment (ykuzmenko73) [inline lib/form/paragraphtypes.flow:6]: move it to the end of export section

**Review by ykuzmenko73 — COMMENTED** [review]
LGTM

**Review by ykuzmenko73 — APPROVED** [review]

**Pull request merged**